pages/studentAssessmentExam/studentAssessmentExam.ejs

```diff
@@ -1,75 +1,180 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <%- include('../partials/head'); %>
-  </head>
-  <body>
-    <%- include('../partials/navbar', {navPage: ''}); %>
-    <div id="content" class="container">
-      <div class="card mb-4">
-        <div class="card-header bg-primary text-white">
-          <%= assessment_set.abbreviation %><%= assessment.number %>: <%= assessment.title %>
-        </div>
-
-        <div class="card-body">
-          <% if (authz_result.mode == 'Exam' || authz_result.mode == 'SEB' || authz_result.password != null ) { %>
-          <p class="lead text-center">
-            Please wait until instructed to start by a proctor
-          </p>
-          <% } %>
 
-          <p class="lead text-center">
-            This is <strong><%= assessment_set.name %> <%= assessment.number %>: <%= assessment.title %></strong> for <strong><%= course.short_name %></strong>
-          </p>
+<head>
+  <%- include('../partials/head'); %>
+</head>
```

Comment:
Why is the indentation changed here? It makes it very hard to review where there are spurious changes showing up in the diff.